Include/tesselator.h

```diff
@@ -138,6 +138,11 @@ typedef struct TESSalloc TESSalloc;
 
 #define TESS_NOTUSED(v) do { (void)(1 ? (void)0 : ( (void)(v) ) ); } while(0)
 
+// Value that avoids loss-of-precision or overflow issues when creating distinct
+// sentinel edges in InitEdgeDict.
+#define TESS_MAX_VALUE ((TESSreal) (2<<23))
```

Comment:
I think that should be 1<<23 (I may have used 2 in earlier comment).

I think this needs more user facing comment, and const names. I suggest `TESS_MAX_VALID_INPUT_VALUE`.
```
These two constants define the valid input coordinate range the library is able to operate on. If any input coordinate is out of range, `tessTesselate()` will fail. The user is expected to handle out of range coordinates (e.g. clamp or discard points).
```